mirdata/medleydb_melody.py

```diff
@@ -69,7 +69,7 @@ class Track(object):
         title (str): title of the track
         genre (str): genre of the track
         is_excerpt (bool):
-        is_instrumental (bool)
+        is_instrumental (bool):
         n_sources (int):
         melody1 (F0Data):
```

Comment:
Melody 1: the pitch of the single most predominant source (often the voice)
Melody 2: the pitch of the predominant source for each point in time
Melody 3: the pitch of any melodic source. Allows for more than one f0 value at a time.